.env.rc1

```diff
@@ -27,6 +27,8 @@ BLOCKSCOUT_SUBNETWORK_NAME="Celo"
 BLOCKSCOUT_METADATA_CRAWLER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celo-monorepo"
 BLOCKSCOUT_METADATA_CRAWLER_IMAGE_TAG="metadata-crawler-007f29dc8ebf837543e6e28a14818076e88c69cd"
 BLOCKSCOUT_METADATA_CRAWLER_SCHEDULE="0 */2 * * *"
+BLOCKSCOUT_OVERRIDE_RPC_ENDPOINT="http://rc1-fullnodes-rpc.rc1.svc.cluster.local:8545"
```

Comment:
It looks like session affinity is not configured for this service, was that intentional?

Reply:
True, thanks for noticing.